setup.py

```diff
@@ -32,7 +32,7 @@
     },
     author='Uber AI Labs',
     author_email='fritzo@uber.com',
-    python_requires=">=3.6",
+    python_requires=">=3.5",
```

Comment:
Could you also update the `Programming Language :: Python :: 3.6` classifier below?